cmd/arc_summary/arc_summary3

```diff
@@ -221,6 +222,28 @@ elif sys.platform.startswith('linux'):
 
         return descs
 
+def handle_unraisableException(exc_type, exc_value=None, exc_traceback=None,
+                               err_msg=None, object=None):
+   handle_Exception(exc_type, object, exc_traceback)
+
+def handle_Exception(ex_cls, ex, tb):
+    if ex is IOError:
```

Comment:
hm, do you have some way to trigger this? i tried and tried, but py3 never gave me IOError(EPIPE) and only gave me BrokenPipeError (unlike py2)

Reply:
I absolutely do not.

Reply:
hm, yeah, okay, so in that case i think we should drop the IOError branch. Consider [Python 3's BrokenPipeError](https://docs.python.org/3/library/exceptions.html#BrokenPipeError):
> A subclass of ConnectionError, raised when trying to write on a pipe while the other end has been closed, or trying to write on a socket which has been shutdown for writing. Corresponds to errno EPIPE and ESHUTDOWN.

And compare it to [Python 2's IOError](https://docs.python.org/2/library/exceptions.html#exceptions.IOError):
> Raised when an I/O operation (such as a print statement, the built-in open() function or a method of a file object) fails for an I/O-related reason, e.g., “file not found” or “disk full”.
>
> This class is derived from EnvironmentError.
> The base class for exceptions that can occur outside the Python system: IOError, OSError. When exceptions of this type are created with a 2-tuple, the first item is available on the instance’s errno attribute (it is assumed to be an error number), and the second item is available on the strerror attribute (it is usually the associated error message). The tuple itself is also available on the args attribute.

And, notably,
> The following exceptions are kept for compatibility with previous versions; starting from Python 3.3, they are aliases of OSError.
> exception EnvironmentError
> exception IOError

So if I'm reading this right, IOError(EPIPE) is a strictly py2 construct, and py3 will only ever raise BrokenPipeError in this case

Reply:
Sounds logical to me. Done.

Thanks for doing so much digging!